crates/hc_sandbox/src/calls.rs

```diff
@@ -543,13 +544,22 @@ pub async fn install_app_bundle(cmd: &mut CmdRunner, args: InstallApp) -> anyhow
     let installed_app = cmd.command(r).await?;
     let installed_app =
         expect_match!(installed_app => AdminResponse::AppInstalled, "Failed to install app");
-    enable_app(
-        cmd,
-        EnableApp {
-            app_id: installed_app.installed_app_id.clone(),
+
+    match &installed_app.manifest {
+        AppManifest::V1(manifest) => match manifest.membrane_proofs_deferred {
+            false => {
+                enable_app(
+                    cmd,
+                    EnableApp {
+                        app_id: installed_app.installed_app_id.clone(),
+                    },
+                )
+                .await?
+            }
+            true => (),
```

Comment:
Suggest using `AppManifest::V1(manifest) if !manifest.membrane_proofs_deferred =>` in the match to collapse the inner match

Reply:
agreed :+1:  - pushed a commit with that change (if I understood it correctly)

Reply:
Well actually that's not quite what I meant but it's tidier again than what I was suggesting anyway so I'm happy :D Updated code looks good!

Reply:
I was after a match guard, I just didn't know they were called that :) https://doc.rust-lang.org/reference/expressions/match-expr.html#match-guards